### DISCUSSION TITLE
2022-03-30 Weekly Meeting 49 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#203] [Weekly Meetings]
Weekly Meeting 49 is happening on Wednesday March 30th, 2022 from 8:30 AM PDT(15:30 UTC) - 10:30 AM PDT (17:30 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
This will be a 2 hour part 2 of the workshop addressing OMI structure and leadership.

**Comment by mrmetaverse** [comment]
>Lets remind each working group to review their meeting times are accurately reflected on the omigroup readme, their working groups readme, AND the omigroup website calendar. :point_down: 

AC: 

- :book: omigroup/omigroup readme accurately reflects time
- :scroll:  each teams omigroup/workinggroup README accurately reflects the time
- :calendar: the OMI Calendar on our website accurately reflects the time

source: https://github.com/omigroup/omigroup/issues/202#issuecomment-1081981777

**Reply by mrmetaverse** [reply]
due to the nature of todays workshop, we may want to bump this topic to next week
